src/Components/UI/Modal/Login.jsx

```diff
@@ -0,0 +1,108 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useModal } from "../../../Context";
+import Button from "../Button/Button";
+import InputTypeOne from "../Input/InputTypeOne";
+import "./Login.css";
+
+const Login = () => {
+  const [loginInput, setLoginInput] = useState({
+    email: "",
+    password: "",
+  });
+
+  const { setShowLogin, setShowSignup } = useModal();
+  const navigate = useNavigate();
+
+  const onLoginSubmitHandler = (e) => {
+    e.preventDefault();
+    navigate("/home");
+    setShowLogin(false);
+  };
+
+  const onModalInputHandler = (e) => {
+    const value = e.target.value;
+    const name = e.target.name;
+    setLoginInput((preData) => {
+      return {
+        ...preData,
+        [name]: value,
+      };
+    });
+  };
+
+  return (
+    <>
+      <div
+        className="modal-backdrop"
+        onClick={() => {
+          setShowLogin(false);
+        }}
+      ></div>
+      <div className="signin-modal">
+        <h1>Welcome Back</h1>
+        <p>Enter your credentials to access your account</p>
+        <a
+          onClick={() => {
+            setShowLogin(false);
+            setShowSignup(false);
+          }}
+        >
+          <i className="fas fa-times"></i>
+        </a>
+        <form onSubmit={onLoginSubmitHandler}>
+          <InputTypeOne
+            type="email"
+            name="email"
+            // required="required"    commented for development
+            autoComplete="email"
+            placeholder="Enter your email *"
+            iconWrapper="input-icon"
+            icon="far fa-envelope"
+            inputWrapper="outline-email-input"
+            onChange={onModalInputHandler}
+            value={loginInput.email}
+          />
+          <InputTypeOne
+            type="password"
+            name="password"
+            // required="required"     commented for development
+            autoComplete="current-password"
```

Comment:
autoCompletes should always be avoided for passwords as best practice

Reply:
It's for a better user experience, as it will take the saved password at the time of login so the user will not have to type it all the time.